indico/MaKaC/common/info.py

```diff
@@ -395,7 +395,7 @@ def isCorrectStyle(self, styleId):
             return False
 
         correctCSS = self.existsCSSFile(styleId) or not self.getCSSFilename(styleId)
-        return styleId == 'static' or (self.existsTPLFile(styleId) and correctCSS) or (self.existsXSLFile(styleId) and correctCSS)
+        return styleId == 'static' or correctCSS or self.existsXSLFile(styleId)
```

Comment:
Do we need to keep the xsl-based styles and thus the xml generation around for now?